test/command_line/test_import.py

```diff
@@ -474,6 +471,36 @@ def test_import_still_sequence_as_experiments_subset(dials_data, tmpdir):
     assert all(exp.goniometer is not None for exp in imported_exp)
 
 
+def test_import_still_sequence_as_expts_subset_by_range(dials_data, tmpdir):
+    image_files = dials_data("centroid_test_data").listdir("centroid*.cbf", sort=True)
+
+    out = "experiments_as_still.expt"
+
+    _ = procrunner.run(
+        [
+            "dials.import",
+            "scan.oscillation=10,0",
+            "image_range=3,5",
+            "output.experiments=%s" % out,
+        ]
+        + [f.strpath for f in image_files],
+        working_directory=tmpdir.strpath,
+    )
+
+    imported_exp = load.experiment_list(tmpdir.join(out).strpath)
+    assert len(imported_exp) == 3
+    for exp in imported_exp:
+        assert exp.identifier != ""
+
+    iset = set(exp.imageset for exp in imported_exp)
+    assert len(iset) == 1
+    assert len(imported_exp[0].imageset) == 3
```

Comment:
My one concern with this test is that it only tests that we only import 3 images, not which 3 images are imported. I.e. we don't explicitly check that we have imported images 3-5 rather than 1-3 (just thinking of ways in which the code could break, yet the tests still pass).

Perhaps checking the summed counts for each image in the imported imageset would suffice (assuming that each image has different total counts)?

Reply:
This point is a good point

Reply:
```
assert list(iset)[0].get_image_identifier(0) == image_files[2].strpath
```

seemed simpler